src/main/scala/inox/ast/TreeOps.scala

```diff
@@ -77,7 +77,7 @@ trait TreeOps { self: Trees =>
       case sort: ADTSort =>
         traverse(sort.id)
         sort.tparams.foreach(traverse)
-        sort.cons.foreach(traverse)
+        sort.cons.sortBy(_.name).foreach(traverse)
```

Comment:
This is already a Seq, so the traversal order is deterministic. This order is used pretty much everywhere so it would be a bit weird to change it here.